README.md

```diff
@@ -32,8 +32,8 @@ This C# SDK is automatically generated by the [Swagger Codegen](https://github.c
 - SDK version: 2.3.1
 - Build package: io.swagger.codegen.v3.generators.dotnet.CSharpClientCodegen
 
-<a name="frameworks-supported"></a>
-## Frameworks supported
+<a name="requirements"></a>
+## Requirements
```

Comment:
Why list Requirements and Dependencies separately? Isn't it all "stuff you need to run this"?

Reply:
These are dependencies used by the package, so they are installed automatically. They are defined here just for the reference.

Reply:
Would the distinction be obvious to a C# beginner? If not, consider adding some clarification like `(installed automatically when adding the package)` under "Dependencies"

Reply:
Done!